src/main/java/dk/cngroup/lentils/service/TeamService.java

```diff
@@ -0,0 +1,78 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.repository.TeamRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.util.Optionals;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+
+public class TeamService {
+
+    TeamRepository repository;
+
+    @Autowired
+    public TeamService(TeamRepository repository) {
+        this.repository = repository;
+    }
+
+    public Team add(Team team) {
+        team.setPin(getUniquePin());
```

Comment:
Idealni bude vytvorit nejakou facrtory, ktera to vytvori a bude obsahovat i tu funkci pro vypocitani nahodneho pinu. Tak aby to nebylo v konstruktoru.